apps/health_campaign_field_worker_app/lib/widgets/member_card/member_card.dart

```diff
@@ -175,7 +175,7 @@ class MemberCard extends StatelessWidget {
                         icon: Icons.info_rounded,
                         iconSize: 20,
                         iconText: localizations.translate(
-                          (isNotEligible || isBeneficiaryIneligible)
+                          isHead ? i18.householdOverView.householdOverViewHouseholderHeadLabel : (isNotEligible || isBeneficiaryIneligible)
```

Comment:
We also need to hide the action buttons when it is a head, lets confirm this as well @amitverma-egov

Reply:
done